frontends/common/parser_options.cpp

```diff
@@ -454,19 +453,22 @@ void ParserOptions::closePreprocessedInput(FILE *inputStream) const {
 
 // From (folder, file.ext, suffix)  returns
 // folder/file-suffix.ext
-static cstring makeFileName(cstring folder, cstring name, cstring baseSuffix) {
-    Util::PathName filename(name);
-    Util::PathName newName(filename.getBasename() + baseSuffix + "." + filename.getExtension());
-    auto result = Util::PathName(folder).join(newName.toString());
-    return result.toString();
+static std::filesystem::path makeFileName(const std::filesystem::path &folder,
```

Comment:
Folder? What are we, Windows programmers? 🤣 

(No change needed -- you're not introducing a new name.)

Reply:
Yeah, likely the code was copied from somewhere else :)